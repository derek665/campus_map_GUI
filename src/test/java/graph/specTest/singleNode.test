# Tests that a graph can be created, a single node can be added, and can be printed out

# Create a graph
CreateGraph g1

# Add a single node
AddNode g1 n1

# Print node in graph
ListNodes g1