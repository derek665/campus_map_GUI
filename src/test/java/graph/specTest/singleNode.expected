# Tests that a graph can be created, a single node can be added, and can be printed out

# Create a graph
created graph g2

# Add a single node
added node n1 to g2

# Print node in graph
g2 contains: n1